README.md

```diff
@@ -1,6 +1,6 @@
 # icon.community – The Official ICON Community Website
 
-The icon.community website is the main hub to direct members of the ICON ecosystem to available resources.
+The icon.community website is the main hub to direct members of the ICON ecosystem to available resources. This site includes blogs, links to other media, and general information about the ICON community.
```

Comment:
This is too generic. Please re-write to tickle the imagination.

Reply:
I fixed it